tools/install/_common.sh

```diff
@@ -60,11 +60,16 @@ function common::install_from_gh_release {
 
   # Download tool
   local -r RELEASES="https://api.github.com/repos/${GH_ORG}/${TOOL}/releases"
+  local CURL_OPTS=()
+
+  [[ "$GITHUB_TOKEN" ]] && CURL_OPTS+=('-H' "Authorization: Bearer $GITHUB_TOKEN")
```

Comment:
Double quotes in double square brackets aren't necessary:
```suggestion
  [[ $GITHUB_TOKEN ]] && CURL_OPTS+=('-H' "Authorization: Bearer $GITHUB_TOKEN")
```